application/features/bootstrap/EmailContext.php

```diff
@@ -1193,35 +1211,19 @@ public function iSendPasswordExpiredEmails()
     }
 
     /**
-     * @Given a mfaManagerBcc email address is configured
-     */
-    public function aMfamanagerbccEmailAddressIsConfigured()
-    {
-        \Yii::$app->params['mfaManagerBcc'] = 'email@example.com';
```

Comment:
One of our IDPs _might_ be using this "bcc" feature. I'm checking on that...

Reply:
Looks like it is.

Reply:
This is fine though as it is replaced by `aParamEmailAddressIsConfigured`, we could roll it back as it only replaces two functions instead of four.